cypress-tests/cypress/support/utils/button.js

```diff
@@ -1,103 +1,39 @@
-import { buttonSelector } from "Selectors/button";
+import { commonWidgetSelector } from "Selectors/common";
 import { buttonText } from "Texts/button";
-import {commonSelectors} from "Selectors/common";
-import { commonText } from "Texts/common";
-
-
-
-export const propertiesElements = () => {
-  cy.get(buttonSelector.buttonWidget).dblclick();
-
-  cy.get(buttonSelector.buttonProperties).should("be.visible").and("have.text",buttonText.buttonProperties);
-  cy.get(buttonSelector.buttonName).should("be.visible");
-
-  cy.get(buttonSelector.propertiesElements.eventsAccordion).click();
-  cy.get(buttonSelector.propertiesElements.layoutAccordion).click();
-  for(const elements in buttonSelector.propertiesElements){
-    cy.get(buttonSelector.propertiesElements[elements]).should("be.visible").and("have.text",buttonText.propertiesElements[elements]);
-  }
-
-  cy.get(buttonSelector.loadingStateToggle).should("be.visible");
-  cy.get(buttonSelector.loadingStateFx).should("be.visible").and("have.text",buttonText.fxButton);
-  cy.get(buttonSelector.desktopToggle).should("be.visible");
-  cy.get(buttonSelector.desktopFx).should("be.visible").and("have.text",buttonText.fxButton);
-  cy.get(buttonSelector.mobileToggle).should("be.visible");
-  cy.get(buttonSelector.mobileFx).should("be.visible").and("have.text",buttonText.fxButton);
-
-};
-
-export const stylesElements = () => {
-  cy.get(buttonSelector.buttonWidget).dblclick();
-  cy.get(buttonSelector.buttonStyles).should("be.visible").and("have.text",buttonText.buttonStyles);
-  cy.get(buttonSelector.buttonStyles).click();
-  for(const elements in buttonSelector.stylesElements){
-    cy.get(buttonSelector.stylesElements[elements]).should("be.visible").and("have.text",buttonText.stylesElements[elements]);
-  }
-  for(const inputs in buttonSelector.stylesInput){
-    cy.get(buttonSelector.stylesInput[inputs]).should("be.visible");
-  }
-  for(const fx in buttonSelector.stylesFx){
-    cy.get(buttonSelector.stylesFx[fx]).should("be.visible").and("have.text",buttonText.fxButton);
-  }
-  cy.get(buttonSelector.visibilityToggle).should("be.visible");
-  cy.get(buttonSelector.disableToggle).should("be.visible");
-
-};
-
-export const deleteApp = () => {
-  cy.get(commonSelectors.autoSave, { timeout: 9000 }).should("have.text", commonText.autoSave);
-
-  cy.get(commonSelectors.backButton).click();
-  cy.get(commonSelectors.appCardOptions).first().click();
-  cy.get(commonSelectors.deleteApp).click();
-  cy.get(commonSelectors.confirmButton).click();
+import { openAccordion, openEditorSidebar } from "Support/utils/commonWidget";
 
+export const openButtonStylesEditorSideBar = (widgetText, widgetName) => {
```

Comment:
I think we can use the common methods to open the sidebar by modifying the `data-cy` of the button.